extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/JdbcBasePersistenceImpl.java

```diff
@@ -122,7 +122,7 @@ public void writeEntity(
         }
       } catch (SQLException e) {
         throw new RuntimeException(
-            String.format("Failed to write entity due to %s", e.getMessage()));
+            String.format("Failed to write entity due to %s", e.getMessage()), e);
```

Comment:
I'd recommend to introduce a new exception for JDBC implementation instead of using the general one `RuntimeException`. We may name it like `JdbcPersistenceException`, which extend `RuntimeException`.